addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_data.xml

```diff
@@ -1,603 +1,1516 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <!--
+    copyright：ps cloud
+    author：mypscloud
+    科目表：2017小企业会计准则
+    科目表依据：http://kjs.mof.gov.cn/zhengwuxinxi/zhengcefabu/201111/t20111107_605525.html
+    -->
 
-<!--
-
-     Copyright (C) 2012-2012 南京盈通 ccdos@intoerp.com <small business chart>
-     
-会计科目表模板( 小企业会计准则2011)
-
-科目表依据：
-关于印发《小企业会计准则》的通知
-http://kjs.mof.gov.cn/zhengwuxinxi/zhengcefabu/201111/t20111107_605525.html
-
--->
+    <!--科目表-->
     <data noupdate="1">
```

Comment:
it is better for translate account data form xml format to csv like  addons ` l10n_cn_standard`.

Reply:
we'll do that on version 12 and master